src/full_node/bundle_tools.py

```diff
@@ -14,6 +14,9 @@ def best_solution_program(bundle: SpendBundle) -> SerializedProgram:
     """
     r = []
     for coin_solution in bundle.coin_solutions:
-        entry = [coin_solution.coin.name(), [coin_solution.puzzle_reveal, coin_solution.solution]]
+        entry = [
+            [coin_solution.coin.parent_coin_info, coin_solution.coin.amount],
```

Comment:
Note that `to` operates differently on `[a, b]` (a list of two elements) and `(a, b)`(a 2-tuple)... we get `(a b)` or `(a . b)`. I think we should consider the second, since it gives us minimal cons boxes without any nils.